src/components/shared/SampleTooltipContents.js

```diff
@@ -38,6 +38,7 @@ type RestProps = {|
 type Props = {|
   +cpuRatioInTimeRange: CPUProps | null,
   ...RestProps,
+  +sampleIndex: IndexIntoSamplesTable | null,
```

Comment:
optional nit: for more consistency, maybe you could move `cpuRatioInTimeRange` below as well?